Dockerfile

```diff
@@ -0,0 +1,38 @@
+# syntax=docker/dockerfile:1
+
+ARG NODE_VERSION=20.12.2
+
+FROM node:${NODE_VERSION}-alpine as base
```

Comment:
Instead of specifying the argument for the node version, can we hard code it since it is used only once?